composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-		"php": ">=7.2.5",
+		"php": ">=7.2.5|^8.0|^8.1",
```

Comment:
What's the issue with the expression ">=7.2"?

Reply:
It is the same Version String as in inspector-laravel package, wasn't it?

Reply:
Yes, but it is not necessary. The expression ">=7.2" includes also the version 8.

Reply:
Ok - then i will close this pr